configs/genesis-assets/genesis.json

```diff
@@ -132,6 +132,42 @@
       "0x01d53d50d204842575c87f25161248aadb26d33c7375d0f043e2a3f1243cf874"
     ]
   ],
+  "predeployed_accounts": [
+    {
+      "contract_address": "0x1",
+      "class_hash": "0x0279d77db761fba82e0054125a6fdb5f6baa6286fa3fb73450cc44d193c2d37f",
+      "name": "No Validation Account",
+      "private_key": null
+    },
+    {
+      "contract_address": "0x2",
+      "class_hash": "0x06f0d6f6ae72e1a507ff4b65181291642889742dbf8f1a53e9ec1c595d01ba7d",
+      "name": "Argent Account",
+      "private_key": [
+        48, 120, 48, 48, 99, 49, 99, 102, 49, 52, 57, 48, 100, 101, 49, 51, 53,
+        50, 56, 54, 53, 51, 48, 49, 98, 98, 56, 55, 48, 53, 49, 52, 51, 102, 51,
+        101, 102, 57, 51, 56, 102, 57, 55, 102, 100, 102, 56, 57, 50, 102, 49,
+        48, 57, 48, 100, 99, 98, 53, 97, 99, 55, 98, 99, 100, 49, 100
+      ]
+    },
+    {
+      "contract_address": "0x3",
+      "class_hash": "0x006280083f8c2a2db9f737320d5e3029b380e0e820fe24b8d312a6a34fdba0cd",
+      "name": "Openzeppelin Account",
+      "private_key": [
+        48, 120, 48, 48, 99, 49, 99, 102, 49, 52, 57, 48, 100, 101, 49, 51, 53,
+        50, 56, 54, 53, 51, 48, 49, 98, 98, 56, 55, 48, 53, 49, 52, 51, 102, 51,
+        101, 102, 57, 51, 56, 102, 57, 55, 102, 100, 102, 56, 57, 50, 102, 49,
+        48, 57, 48, 100, 99, 98, 53, 97, 99, 55, 98, 99, 100, 49, 100
```

Comment:
Shouldn't that be as hex string ?

Reply:
[Apparently it shouldn't](https://github.com/keep-starknet-strange/madara/pull/1265#discussion_r1404083204)

Reply:
hex string is different than field element.
But why do you prefer an hexstring @EvolveArt ? Just more handy to copy-paste?